gobcore/logging/logger.py

```diff
@@ -223,3 +256,11 @@ def method(*args, **kwargs):
 
 
 logger = LoggerManager()
+
+# Use our ExtendedLogger with data logging as the default logger
+logging.setLoggerClass(ExtendedLogger)
+
+# Add custom logging levels for Data info, warnings and errors to distinguish between data and process logging
+logging.addLevelName(ExtendedLogger.DATAINFO, 'DATAINFO')
+logging.addLevelName(ExtendedLogger.DATAWARNING, 'DATAWARNING')
+logging.addLevelName(ExtendedLogger.DATAERROR, 'DATAERROR')
```

Comment:
Can we not put this code in a ExtendedLogger class method called initialize so that all ExtendedLogger specific code is concentrated in the class. If anytime a new logging logging level should be introduced the maintenance is concentrated in the class and not at 2 places